Code/Tools/ProjectManager/Source/NewProjectSettingsScreen.cpp

```diff
@@ -104,7 +104,45 @@ namespace O3DE::ProjectManager
                     else if (button && button->property(k_addRemoteTemplateProperty).isValid())
                     {
                         AddRemoteTemplateDialog* addRemoteTemplateDialog = new AddRemoteTemplateDialog(this);
-                        addRemoteTemplateDialog->exec();
+                        if (addRemoteTemplateDialog->exec() == QDialog::DialogCode::Accepted)
+                        {
+                            auto remoteTemplatesResult =
+                                PythonBindingsInterface::Get()->GetProjectTemplatesForRepo(addRemoteTemplateDialog->GetRepoPath());
+                            if (remoteTemplatesResult.IsSuccess() && !remoteTemplatesResult.GetValue().isEmpty())
+                            {
+                                // remove remote template button from layout so we can insert the new templates before it
+                                m_templateFlowLayout->removeWidget(m_remoteTemplateButton);
+
+                                int currentTemplateIndex = m_templates.size();
+                                const QVector<ProjectTemplateInfo>& remoteTemplates = remoteTemplatesResult.GetValue();
```

Comment:
Should we sort all the templates alphabetically or will that happen automatically the next time they load this page?

Reply:
Yes, they'll be sorted the next time this page is loaded.